src/reactive/runner.py

```diff
@@ -0,0 +1,32 @@
+#  Copyright 2024 Canonical Ltd.
+#  See LICENSE file for licensing details.
+
+"""Module which contains code to spawn a runner reactively."""
+import logging
+
+from reactive.job import Job, JobError, MessageQueueConnectionInfo
+
+logger = logging.getLogger(__name__)
+
+
+def reactive_runner(mq_uri: str, queue_name: str) -> None:
```

Comment:
Nit:
a little suggestion for naming :D
```suggestion
def spawn_reactive_runner(mq_uri: str, queue_name: str) -> None:
```